vector/vector.toml

```diff
@@ -1,4 +1,4 @@
-# Vector Sample Config
+# Vector Sample Configuration for defaults
```

Comment:
I think this whole file should be replaced with defaults that actually function. I do not like the idea of having some dummy data showing up in a cluster. It should either be real data, or it should exit.

EG, this snip that was generated using `dasel -r yaml -f foo.toml -w toml` against the `vector-config.yaml` file found in <https://github.com/astronomer/airflow-chart/pull/312/files#diff-06265d7d34eecf75ee03e54e085ff4cf55fd01b13339ec14ba2d9e19cffd7002R16>

```
data_dir = "${SIDECAR_LOGS}"

[sinks]

  [sinks.out]
    compression = "none"
    inputs = ["transform_syslog"]
    mode = "bulk"
    type = "elasticsearch"

[sources]

  [sources.generate_syslog]
    include = ["${SIDECAR_LOGS}/*.log"]
    read_from = "beginning"
    type = "file"

[transforms]

  [transforms.transform_syslog]
    inputs = ["generate_syslog"]
    type = "add_fields"

    [transforms.transform_syslog.fields]
      component = "${COMPONENT:--}"
      release = "${RELEASE:--}"
      workspace = "${WORKSPACE:--}"
```

Reply:
let me test with this config locally

Reply:
@pgvishnuram any word on this?

Reply:
This was fixed in another PR.